youtube_dl/extractor/tvanouvelles.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class TVANouvellesIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.|)tvanouvelles\.(ca|com|qc)/.*.?/(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'http://www.tvanouvelles.ca/videos/5117035533001',
+        'info_dict': {
+            'id': '5117035533001',
+            'ext': 'mp4',
+            'title': 'L\u2019industrie du taxi d\xe9nonce l\u2019entente entre Qu\xe9bec et Uber: explications',
```

Comment:
It's UTF-8 and my editor was complaining because Python if I understand correctly is ASCII. I would of had to set it as UTF-8. Should have I done that?